data/webapi/endpoints/calendar.yaml

```diff
@@ -3,5 +3,15 @@ endpoints:
     GET:
       title: Get the calendar
       summary: Returns the market calendar.
+      params:
+        query:
+          - name: start
+            type: timestamp
+            desc: |
+              The first date to retrieve data for
+          - name: end
+            type: timestamp
+            desc: |
+              The last date to retrieve data for
```

Comment:
Good catch. Is it inclusive or exclusive boundary?

Reply:
I don't know

Reply:
updated @umitanuki

Reply:
thx